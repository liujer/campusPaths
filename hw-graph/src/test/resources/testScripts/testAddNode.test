# Tests adding three nodes to a directed graph, as well as
# adding duplicate nodes to another directed graph
CreateGraph A
AddNode A one
AddNode A two
AddNode A three
ListNodes A
CreateGraph B
AddNode B one
AddNode B two
AddNode B two
ListNodes B