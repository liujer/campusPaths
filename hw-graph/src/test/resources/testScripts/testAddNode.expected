# Tests adding three nodes to a directed graph, as well as
# adding duplicate nodes to another directed graph
Created graph A
Added node one to A
Added node two to A
Added node three to A
ListNodes of A output: one three two
Created graph B
AddNode B one
Added node one to B
Added node two to B
Could not add node two to B
ListNodes of B output: one two